app/views/docs/functions-develop.phtml

```diff
@@ -444,10 +481,10 @@ public class Handler {
 </p>
 
 <p>You'll find these properties in the context object.</p>
-<table class="full text-size-small">
+<table cellspacing="0" cellpadding="0" border="0"  class="full text-size-small">
```

Comment:
why do we need to add this?

Reply:
I thought this was causing the extra column to be added there... but it was not . removed it now